mkdocs.yml

```diff
@@ -84,6 +84,7 @@ plugins:
             - jax.extend.core.ClosedJaxpr
             - jax.Device
             - jax.sharding.Sharding
+            - wadler_lindig.AbstractDoc
```

Comment:
This line probably isn't necessary any more?

Reply:
True!